github-actions/oss-readme-template/src/main.ts

```diff
@@ -35,20 +35,27 @@ function run(): void {
       trimWhitespace: true,
     });
 
+    const usageExamplePath = core.getInput('usage_example_path', {
```

Comment:
For now I think this field is only going to be used for SDKs (not our other OSS projects), so let's move this into the `getProjectInfo` function, and make it a required property for SDKs.  It should end up looking almost exactly the same as the `sdkLanguage` one.